tests/unit/charms/lte_core_interface/v0/test_lte_core_interface_requirer.py

```diff
@@ -60,6 +60,22 @@ def test_given_valid_lte_core_information_in_relation_data_when_relation_changed
 
         patch_on_lte_core_available.assert_called()
 
+    @patch(f"{BASE_CHARM_PATH}._on_lte_core_available")
+    def test_given_lte_core_information_not_in_relation_data_when_relation_changed_and_schema_validation_fails_then_core_available_event_not_emitted(  # noqa: E501
```

Comment:
Are we testing the info is not in relation data?

Reply:
✅